### PR TITLE
Uses the faster template/fn instead of template/eval

### PR DESCRIPTION
**Pull request by shamsimam**
## Changes proposed in this PR

- Uses the faster template/fn instead of template/eval

## Why are we making these changes?

- avoids repeated calls to eval (which requires repeated compilation)

### CODE REVIEW

File: waiter/src/waiter/handler.clj [hunk 1]
```diff
@@ -715,21 +724,29 @@
...
       (meters/mark! (metrics/waiter-meter "auto-run-as-requester" "form-error"))
       (utils/exception->response ex request))))
 
+(let [html-fn (template/fn [{:keys [cid host hostname message port support-info timestamp]}]
+                           (slurp (io/resource "web/welcome.html")))]
+  (defn- render-welcome-html "Renders welcome html" [context] (html-fn context)))
+
+(let [text-fn (template/fn [{:keys [cid host hostname message port support-info timestamp]}]
+                           (slurp (io/resource "web/welcome.txt")))]
+  (defn- render-welcome-text "Renders welcome text" [context] (text-fn context)))
```
Comment (dposada) [inline waiter/src/waiter/handler.clj]: These `defn`s where everything is on the same line look odd to me

Comment (shamsimam) [reply waiter/src/waiter/handler.clj]: Done.

**Comment by shamsimam** [comment]
> Do we know if this helps overall performance?

I am going to test this today.

**Comment by shamsimam** [comment]
Stress tested and this change looks good. I will rebase https://github.com/twosigma/waiter/pull/192 after this is merged to avoid generating exceptions while rendering the error messages.

**Pull request merged**